Add StaticList to lab (#4194)
jpmorganchase/salt-ds: site/docs/components/static-list/accessibility.mdx

---
title:
  $ref: ./#/title
layout: DetailComponent
sidebar:
  exclude: true
data:
  $ref: ./#/data
---

## Best practices

> You'll need to check and test any focusable elements included in the static list content accordingly.

Reviewer: ```suggestion
You must check and test any focusable elements inside the static list content accordingly.
```
?

Author: sounds better, will update toast too to avoid replicating this wording in the future.
